crates/re_space_view_spatial/src/parts/boxes2d.rs

```diff
@@ -21,15 +21,56 @@ use super::{
     process_colors, process_radii, SpatialViewPartData,
 };
 
-pub struct Boxes2DPart(SpatialViewPartData);
+pub struct Boxes2DPart {
+    /// If the number of points in the batch is > max_labels, don't render box labels.
+    pub max_labels: usize,
```

Comment:
I think actually we should drop this weirdness; user has zero control over that stuff. If they spam us with labels we should just spam back.
In fact I believe we only ever did this for points

I'll take care of it in a follow-up, don't worry about it here